osu.Game/Screens/Select/SearchTextBox.cs

```diff
@@ -3,19 +3,20 @@
 
 using System;
 using System.Linq;
-using OpenTK;
 using OpenTK.Graphics;
 using OpenTK.Input;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Primitives;
-using osu.Framework.Graphics.Sprites;
-using osu.Framework.Graphics.UserInterface;
 using osu.Framework.Input;
 using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterface;
 
 namespace osu.Game.Screens.Select
 {
-    public class SearchTextBox : TextBox
+    /// <summary>
+    /// A textbox which holds focus eagerly.
+    /// </summary>
+    public class SearchTextBox : OsuTextBox

```

Comment:
Does the name still make sense then? I mean, the only thing related to search is the font-awesome icon. Since we might need a different style for the options search anyway, FocusedTextBox and SearchTextBox can likely be separated.